src/components/ButtonReport.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from '@chakra-ui/react';
+
+export type ButtonReportProps = {
+  onClick: () => void;
+  text: string;
+  title: string;
+  disabled: boolean;
```

Comment:
here is an example of props that could be sorted by alpha

Reply:
Done :)